x

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./build.sh --force && cd /Users/avacher/code/hackday/opensource/metube-browser-extension/builds && unzip -o metube-browser-extension-1.3.zip -d metube-browser-extension-1.3; cd ~/code/hackday/opensource/metube-browser-extension
```

Comment:
you can add that file into local .gitignore rules or just not commit it. we don't need it into repo

Reply:
Yeah, I incorrectly committed this - removing